packages/driver/src/cy/big_value.ts

```diff
@@ -0,0 +1,149 @@
+import _ from 'lodash'
+import $dom from '../dom'
+
+const LOGGED_ARRAY_SIZE = 10
+const LOGGED_OBJ_SIZE = 10
```

Comment:
I see these consts defined in `chai.js` as well. Is this intended?

Reply:
I intended to place `big_value` code in `chai.js`, but decided not to. I forgot to remove them.